app/api/v2/views/authentication.py

```diff
@@ -37,70 +34,67 @@ def post(self):
         password = generate_password_hash(data['password'])
 
         try:
-            access_token = create_access_token(identity=data['username']),
-            refresh_token = create_refresh_token(identity=data['username'])    
             user = self.db.get_username_user(username)
             if user:
-                return make_response(jsonify({
-                    "message": "User {} already exists".format(data['username'])
-                }))
+                return make_response(
+                    jsonify({
+                        "message":
+                        "User {} already exists".format(data['username'])
+                    }), 404)
+            
             sign_up = self.db.save(firstname, lastname, othernames, username,
-                                email, phonenumber, password)
+                                   email, phonenumber, password)
 
             return make_response(jsonify({
                 "status": 201,
-                "data": [{
-                    "access_token": access_token,
-                    "refresh_token": refresh_token,
-                    "incident_created": sign_up,
-                    "message": "Created {} successfuly, you can now login ".format(username)
-                }]
+                "message":
+                "Created {} successfuly, you can now login ".format(username),
             }), 201)
         except:
             return make_response(jsonify({
-                "message": "Check again"
-            }), 500)
+                "message": "User creation not successful"
+            }), 400)
 
 
 class SignIn(Resource, UsersModel):
     """ Docstring sign in which allows already registered users to sign in """
 
     def __init__(self):
         self.db = UsersModel()
-    
-    
+
     def post(self):
         """ Registered user login and validation """
         login_data = request.get_json(force=True)
         if not login_data:
             return make_response(jsonify({
                 "status": 200,
                 "message": "Kindly input Username and Password details"
-            }), 404)
+            }), 200)
```

Comment:
why are you returning two status codes here?

Reply:
I have two status codes because of the project requirements, they require that I input a status code that way. and the second one is to ensure that it returns the error code to the rest client.